imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js

```diff
@@ -43,10 +44,49 @@ Template.searchModal.onCreated(function () {
     }
   });
 
+    // Filters product by price
+  const priceFilter = (products, query) =>  {
+    return _.filter(products, (product) => {
```

Comment:
No, not all products has both value but what I noticed is that those being set without variants has price set to null and those that has only one variant has both min and max price being allocated automatically to them. So what I did to tackle this is to handle those that has the price set  to null so as to reflect when filtering